src/cpp/r/R/Api.R

```diff
@@ -707,16 +707,14 @@ options(terminal.manager = list(terminalActivate = .rs.api.terminalActivate,
 
 .rs.addApiFunction("getThemeInfo", function() {
    
-   # read theme preferences
+   # read theme preferences (note: 'theme' is deprecated)
    editor <- .rs.readUiPref("theme")
```

Comment:
If you check this value, it may show a stale value for a user who just upgraded and hasn't set their theme yet. I.e. before upgrade they were using Tomorrow Night. After upgrade, it looks like they have Textmate set, but really there's no `rstheme` value in the preferences. Then if they call this function, it will return Tomorrow Night instead of Textmate.

Reply:
Ah, got it. Fixed that up here: https://github.com/rstudio/rstudio/commit/75a3891d744f48a6cba5b228da6d99331163b0d9